### PR TITLE
APB-5492 add new accordion and hmrcLayout

### PR DESCRIPTION
**Pull request by max-hmrc**

### CODE REVIEW

File: app/uk/gov/hmrc/agentservicesaccount/config/AppConfig.scala [hunk 1]
```diff
@@ -93,7 +93,7 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: ServicesConfig
   val vatUrl = s"$vatExternalUrl$vatStartPath"
 
   val agentSuspensionEnabled = getBoolean("features.enable-agent-suspension")
-  val welshToggleEnabled = getBoolean("features.enable-welsh-toggle")
+  val welshToggleEnabled = true
```
Comment (geoffreywatson) [inline app/uk/gov/hmrc/agentservicesaccount/config/AppConfig.scala:96]: why not remove this config?

File: app/uk/gov/hmrc/agentservicesaccount/views/main.scala.html [hunk 2]
```diff
@@ -14,41 +14,32 @@
...
-    <script>
-            window.GOVUKFrontend.initAll();
-            window.HMRCFrontend.initAll();
-    </script>
+    <script src="@controllers.routes.Assets.at("javascripts/jquery.js")" />
+    <script src="@controllers.routes.Assets.at("javascripts/ga-events.js")" />
 }
 
-@govukLayout(
+@hmrcLayout(
```
Comment (CarolineS-QA) [inline app/uk/gov/hmrc/agentservicesaccount/views/main.scala.html:39]: Think we need a wider discussion about switching to hmrcLayout in `agent-services-account-frontend` since this is the one exception to keeping the Welsh toggle in the black part of the header, since it was part of the original ASAF design

File: app/uk/gov/hmrc/agentservicesaccount/config/AppConfig.scala (hunk 1 above)
Comment (max-hmrc) [reply app/uk/gov/hmrc/agentservicesaccount/config/AppConfig.scala:96]: Yes please. I did ask about other configs too.

Comment (max-hmrc) [reply app/uk/gov/hmrc/agentservicesaccount/config/AppConfig.scala]: done

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -1,9 +1,3 @@
-$govuk-assets-path: "/agent-services-account/assets/lib/govuk-frontend/govuk/assets/";
```
Comment (CarolineS-QA) [inline app/assets/stylesheets/application.scss:1]: Everything else looks good. Why is this removed? Is this because of the issues with local styles you were having... Is it not fixed if we are using `hmrcLayout` ?

Comment (max-hmrc) [reply app/assets/stylesheets/application.scss:1]: yes. don't need this now

Comment (CarolineS-QA) [reply app/assets/stylesheets/application.scss:1]: Okay
